### PR TITLE
changed api indexes

### PR DESCRIPTION
**Pull request by joshuajhun**
@rrgayhart

### CODE REVIEW

**Comment by joshuajhun** [comment]
@rrgayhart

File: app/assets/javascripts/ideabox_service.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ $('#index').delegate("#delete-idea", 'click', function(){
   var idea = this.closest('#idea')
     $.ajax({
       type: 'DELETE',
-       url: 'http://socalbros:3000/api/v1/ideas/'+ $(idea).attr('data-id'),
+       url: '/api/v1/ideas/'+ $(idea).attr('data-id'),
```
Comment (rrgayhart) [inline app/assets/javascripts/ideabox_service.js:40]: If you DRY this code up a little bit, you can avoid some of the repetition when it comes to changing it in 7 places!

File: app/assets/javascripts/search.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+function searchIdeas(){
+  $("#search").keyup(function(event){
+    var searchParam = $(this).val().toLowerCase();
+    var allExisitingIdeas = $('#index').children();
+
+    allExisitingIdeas.removeClass('invisible');
+
+    var hideIdeas = allExisitingIdeas.filter(function(){
+    var ideaBodyAndText = $(this).find("#ideaname, #ideabody").text().toLowerCase()
```
Comment (rrgayhart) [inline app/assets/javascripts/search.js:9]: Got some wonky indentation here!

File: app/assets/javascripts/search.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+function searchIdeas(){
+  $("#search").keyup(function(event){
+    var searchParam = $(this).val().toLowerCase();
+    var allExisitingIdeas = $('#index').children();
+
+    allExisitingIdeas.removeClass('invisible');
```
Comment (rrgayhart) [inline app/assets/javascripts/search.js:6]: Just a thought, but would it make sense to filter the ideas first, and then only remove the invisible class from those ideas that should not be hidden?

**Pull request merged**